_build/resolvers/setupoptions.resolver.php

```diff
@@ -78,6 +78,12 @@
             }
             $tv->set('type', $tvInfo['type']);
             $tv->set('caption', $tvInfo['caption']);
+            if (isset($tvInfo['rank'])) {
+                $tv->set('rank', $tvInfo['rank']);
+            }
+            if (isset($tvInfo['input_option_values'])) {
+                $tv->set('elements', $tvInfo['input_option_values']);
```

Comment:
is setting it to `elements` right?

Reply:
Yeah I thought it was strange too... I haven't set input options on a tv programmatically before.
I basically manually created a checkbox tv then looked at which column the value went into. Maybe a legacy naming scheme?

Works fine when installing the package.